api/src/main/java/org/openmrs/module/fhir2/model/FhirDurationUnitMap.java

```diff
@@ -84,6 +84,7 @@
 	private String uuid = UUID.randomUUID().toString();
 	
 	@Override
+	@Deprecated
```

Comment:
Why should we generate a warning?

Reply:
isRetired method is deprecated in core OpenMRS now. That creates warnings during project building. We can not just remove the method, so we need to mark it with `@Deprecated` annotation.

Reply:
Thanks, Some of the models is used for mapping purpose and used in this module only. @ibacher @varung-31 your thought?